internal/cmd/connect/command_plugin_install.go

```diff
@@ -24,18 +24,27 @@ import (
 )
 
 type manifest struct {
+	Name    string `json:"name"`
+	Title   string `json:"title"`
+	Version string `json:"version"`
+}
```

Comment:
This is what I was thinking of!
```suggestion
type manifest struct {
	Owner   owner   `json:"owner"`
	Name    string  `json:"name"`
	Title   string  `json:"title"`
	Version string  `json:"version"`
	Archive archive `json:"archive"
	License license `json:"license"`
}
```